### ISSUE TITLE
master branch build failed

### ISSUE THREAD

**Issue by linuxaged** [mosra/magnum#421]
Platform:
* Windows 10 Pro
* VS 2019

Steps to reproduce:
* `vcpkg install --head magnum`

```
C:\workspace\vcpkg\buildtrees\magnum\src\head\master-6881dd6238\src\Magnum\GL\Implementation\RendererState.cpp(115): error C2440: '=': cannot convert from 'void (__stdcall *)(GLenum,GLint)' to 'void (__cdecl *)(GLenum,GLint)'
C:\workspace\vcpkg\buildtrees\magnum\src\head\master-6881dd6238\src\Magnum\GL\Implementation\RendererState.cpp(115): note: This conversion requires a reinterpret_cast, a C-style cast or function-style cast
C:\workspace\vcpkg\buildtrees\magnum\src\head\master-6881dd6238\src\Magnum\GL\Implementation\RendererState.cpp(133): error C2440: '=': cannot convert from 'void (__stdcall *)(GLenum,GLuint)' to 'void (__cdecl *)(GLenum,GLuint)'
C:\workspace\vcpkg\buildtrees\magnum\src\head\master-6881dd6238\src\Magnum\GL\Implementation\RendererState.cpp(133): note: This conversion requires a reinterpret_cast, a C-style cast or function-style cast
C:\workspace\vcpkg\buildtrees\magnum\src\head\master-6881dd6238\src\Magnum\GL\Implementation\RendererState.cpp(134): error C2440: '=': cannot convert from 'void (__stdcall *)(GLenum,GLuint)' to 'void (__cdecl *)(GLenum,GLuint)'
C:\workspace\vcpkg\buildtrees\magnum\src\head\master-6881dd6238\src\Magnum\GL\Implementation\RendererState.cpp(134): note: This conversion requires a reinterpret_cast, a C-style cast or function-style cast
C:\workspace\vcpkg\buildtrees\magnum\src\head\master-6881dd6238\src\Magnum\GL\Implementation\RendererState.cpp(135): error C2440: '=': cannot convert from 'void (__stdcall *)(GLuint,GLboolean,GLboolean,GLboolean,GLboolean)' to 'void (__cdecl *)(GLuint,GLboolean,GLboolean,GLboolean,GLboolean)'
C:\workspace\vcpkg\buildtrees\magnum\src\head\master-6881dd6238\src\Magnum\GL\Implementation\RendererState.cpp(135): note: This conversion requires a reinterpret_cast, a C-style cast or function-style cast
C:\workspace\vcpkg\buildtrees\magnum\src\head\master-6881dd6238\src\Magnum\GL\Implementation\RendererState.cpp(136): error C2440: '=': cannot convert from 'void (__stdcall *)(GLuint,GLenum,GLenum)' to 'void (__cdecl *)(GLuint,GLenum,GLenum)'
C:\workspace\vcpkg\buildtrees\magnum\src\head\master-6881dd6238\src\Magnum\GL\Implementation\RendererState.cpp(136): note: This conversion requires a reinterpret_cast, a C-style cast or function-style cast
C:\workspace\vcpkg\buildtrees\magnum\src\head\master-6881dd6238\src\Magnum\GL\Implementation\RendererState.cpp(137): error C2440: '=': cannot convert from 'void (__stdcall *)(GLuint,GLenum,GLenum,GLenum,GLenum)' to 'void (__cdecl *)(GLuint,GLenum,GLenum,GLenum,GLenum)'
C:\workspace\vcpkg\buildtrees\magnum\src\head\master-6881dd6238\src\Magnum\GL\Implementation\RendererState.cpp(137): note: This conversion requires a reinterpret_cast, a C-style cast or function-style cast
C:\workspace\vcpkg\buildtrees\magnum\src\head\master-6881dd6238\src\Magnum\GL\Implementation\RendererState.cpp(138): error C2440: '=': cannot convert from 'void (__stdcall *)(GLuint,GLenum)' to 'void (__cdecl *)(GLuint,GLenum)'
C:\workspace\vcpkg\buildtrees\magnum\src\head\master-6881dd6238\src\Magnum\GL\Implementation\RendererState.cpp(138): note: This conversion requires a reinterpret_cast, a C-style cast or function-style cast
C:\workspace\vcpkg\buildtrees\magnum\src\head\master-6881dd6238\src\Magnum\GL\Implementation\RendererState.cpp(139): error C2440: '=': cannot convert from 'void (__stdcall *)(GLuint,GLenum,GLenum)' to 'void (__cdecl *)(GLuint,GLenum,GLenum)'
C:\workspace\vcpkg\buildtrees\magnum\src\head\master-6881dd6238\src\Magnum\GL\Implementation\RendererState.cpp(139): note: This conversion requires a reinterpret_cast, a C-style cast or function-style cast
```

**Comment by williamjcm** [comment]
I get the same error with the `x86-windows` triplet, while `x64-windows` works.

**Comment by mosra** [comment]
Fixed in 2562ca447c4eb45a2bb6a306463ec042e38fca8f. Additionally, to prevent these accidents from appearing again in the future, I added a 32-bit MSVC build to the CI build matrix in f4168086511bd41e5835653dfe1ad35521af81ae.
